src/sentry/static/sentry/app/views/projectAlertRules.jsx

```diff
@@ -221,14 +256,13 @@ const ProjectAlertRules = createReactClass({
   },
 
   render() {
-    let {orgId, projectId} = this.props.params;
     return (
       <div>
         <SettingsPageHeader
           title={t('Alerts')}
           action={
             <Button
-              to={`/${orgId}/${projectId}/settings/alerts/rules/new/`}
+              to={recreateRoute('new/', this.props)}
```

Comment:
Why swap all of these out? Is `recreateRoute()` the way we'll want to construct all routes from now on?

Reply:
No. Its only used in settings until we remove legacy settings. It’s so we can navigate with respect to which settings page you’re on.